include/miopen/miopen.h

```diff
@@ -581,6 +589,8 @@ typedef enum
     MIOPEN_CONVOLUTION_ATTRIB_DETERMINISTIC =
         1, /*!< Restrict MIOpen convolutions to kernels which produce numerically deterministic
               results. 0 - disabled (default), 1 - enabled >*/
+    MIOPEN_CONVOLUTION_ATTRIB_FP8_ROUNDING_MODE = 2,
+    MIOPEN_CONVOLUTION_ATTRIB_COMPUTE_TYPE      = 3,
```

Comment:
The documentation of these two attributes is missing. This is simple: for example, for `MIOPEN_CONVOLUTION_ATTRIB_FP8_ROUNDING_MODE` you can simply say "A value from miopenF8RoundingMode_t". But the glue should be provided.

Reply:
Fixed!